addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -453,6 +462,168 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+    const checkboxUnchecked = document.querySelector('.o_ComposerSuggestedRecipient:not([data-partner-id]) input[type=checkbox]');
+    assert.notOk(
+        checkboxUnchecked.checked,
+        "suggested recipient without a partner should not be checked by default",
+    );
+    const checkboxChecked = document.querySelector('.o_ComposerSuggestedRecipient[data-partner-id] input[type=checkbox]');
+    assert.ok(
+        checkboxChecked.checked,
+        "suggested recipient with a partner should be checked by default.",
+    );
+});
+
+QUnit.test("suggested recipients list should have a show more when there is 4 or more suggestion and show less button otherwise", async function (assert) {
+    assert.expect(5);
+
+    this.data['res.partner'].records.push({
+        display_name: "John Jane",
+        email: "john@jane.be",
+        id: 100,
+    });
+    this.data['res.partner'].records.push({
+        display_name: "Jack Jone",
+        email: "jack@jone.be",
+        id: 1000,
+    });
+    this.data['res.partner'].records.push({
+        display_name: "jolly Roger",
+        email: "Roger@skullflag.com",
+        id: 1001,
+    });
+    this.data['res.fake'].records.push({
+        id: 10,
+        email_cc: "john@test.be",
+        partner_ids: [100, 1000, 1001],
+    });
+    await this.start ();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_ComposerSuggestedRecipientList_showMore',
+        "suggested recipient list should containt a \"show more\" button."
+    );
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ComposerSuggestedRecipientList_showMore`).click()
+    );
+    assert.containsN(
+        document.body,
+        '.o_ComposerSuggestedRecipient',
+        4,
+        "suggested recipient list should display all the suggested recipients."
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_ComposerSuggestedRecipientList_showLess',
+        "suggested recipient list should containt a \"show less\" button."
+    );
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ComposerSuggestedRecipientList_showLess`).click()
+    );
+    assert.containsN(
+        document.body,
+        '.o_ComposerSuggestedRecipient',
+        3,
+        "suggested recipient list should display 3 suggested recipients after clicking on \"show less\"."
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_ComposerSuggestedRecipientList_showMore',
+        "suggested recipient list should containt a \"show More\" button after clicking on \"show less\"."
+    );
```

Comment:
Test is named _"should have [...]"_ but actually also plays with toggle show more/show less. This should be another test by itself.